### PR TITLE
*: update LICENSE header

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/issues/5341.

### CODE REVIEW

File: client/cancelreq.go [hunk 1]
```diff
@@ -1,6 +1,16 @@
-// Copyright 2015 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
+// Copyright 2015 The etcd Authors
```
Comment (xiang90) [inline client/cancelreq.go]: this is copied from go. please do not change its header.

File: clientv3/integration/main_test.go [hunk 2]
```diff
@@ -1,6 +1,16 @@
-// Copyright 2013 The Go Authors. All rights reserved.
```
Comment (xiang90) [inline clientv3/integration/main_test.go]: same here

File: e2e/main_test.go [hunk 3]
```diff
@@ -1,6 +1,16 @@
-// Copyright 2013 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
+// Copyright 2013 The etcd Authors
```
Comment (xiang90) [inline e2e/main_test.go]: same here

File: client/cancelreq.go (hunk 1 above)
Comment (gyuho) [inline client/cancelreq.go]: Yeah just found out. Will revert those.

**Comment by gyuho** [comment]
@xiang90 PTAL.

Only thing that changes is `CoreOS, Inc.` to  `The etcd Authors`.
Everything else stays the same.

**Comment by xiang90** [comment]
lgtm

**Comment by gyuho** [comment]
Thanks. Merging in greens.

**Comment by mitake** [comment]
@gyuho lgtm, thanks for notify!

**Comment by gyuho** [comment]
Np!

**Pull request merged**